src/imap.rs

```diff
@@ -1595,7 +1624,7 @@ pub(crate) async fn prefetch_should_download(
     let is_reply_to_chat_message = parent.is_some();
     if let Some(parent) = &parent {
         let chat = chat::Chat::load_from_db(context, parent.get_chat_id()).await?;
-        if chat.typ == Chattype::Group {
+        if chat.typ == Chattype::Group && !chat.id.is_special() {
```

Comment:
What does this change do?

Reply:
This is an unrelated bugfix that was needed to get the test to run:

Sometimes the parent chat is the deaddrop chat, e.g. as the sender was blocked. The deaddrop chat has type Group, so that `chat.typ == Chattype::Group` returns true.

When we remove the deaddrop chat, I still think this makes sense as the trash chat also is a `Group`.

The test:

```python
        lp.sec("Test that the connectivity doesn't flicker to WORKING if the sender of the message is blocked")
        ac1.create_contact(ac2).block()

        ac1.direct_imap.select_config_folder("inbox")
        ac1.direct_imap.idle_start()
        ac2.create_chat(ac1).send_text("Hi")

        ac1.direct_imap.idle_check(terminate=True)
        ac1.maybe_network()

        ac1._evtracker.wait_for_connectivity_change(const.DC_CONNECTIVITY_INTERRUPTING_IDLE,
                                                    const.DC_CONNECTIVITY_CONNECTED)
```